manifest.toml

```diff
@@ -51,7 +51,7 @@ ram.runtime = "1024M"
 
         [resources.sources.main]
         url = "https://github.com/motioneye-project/motioneye/archive/refs/tags/0.43.1b1.tar.gz"
```

Comment:
usually, Yunohost packages stay with stable versions and avoid pre-releases

Reply:
The stable release in this ocassion is a python2 base build, while this beta release is the python3 rewrite